setup.cfg

```
@@ -93,6 +93,8 @@ plot =
     ipython>=7.13.0,<=7.20.0
 trx =
     trx-python
+ray =
+    ray
```

We could combine these: 

```suggestion
    ray
```

If we go with that, you would have to remove the "ray" option below on L106

---

changed in last commit

---

done!